templates/master/01-master-container-runtime/_base/files/crio.yaml

```diff
@@ -1,262 +1,30 @@
 filesystem: "root"
 mode: 0644
-path: "/etc/crio/crio.conf"
+path: "/etc/crio/crio.conf.d/00-default"
 contents:
   inline: |
-    [crio]
-
-    # The default log directory where all logs will go unless directly specified by
-    # the kubelet. The log directory specified must be an absolute directory.
-    log_dir = "/var/log/crio/pods"
-
-    # Location for CRI-O to lay down the version file
-    version_file = "/var/lib/crio/version"
-
-    # The crio.api table contains settings for the kubelet/gRPC interface.
     [crio.api]
-
-    # Path to AF_LOCAL socket on which CRI-O will listen.
-    listen = "/var/run/crio/crio.sock"
-
-    # Host IP considered as the primary IP to use by CRI-O for things such as host network IP.
-    host_ip = ""
-
-    # IP address on which the stream server will listen.
     stream_address = ""
-
-    # The port on which the stream server will listen.
     stream_port = "10010"
 
-    # Enable encrypted TLS transport of the stream server.
-    stream_enable_tls = false
-
-    # Path to the x509 certificate file used to serve the encrypted stream. This
-    # file can change, and CRI-O will automatically pick up the changes within 5
-    # minutes.
-    stream_tls_cert = ""
-
-    # Path to the key file used to serve the encrypted stream. This file can
-    # change and CRI-O will automatically pick up the changes within 5 minutes.
-    stream_tls_key = ""
-
-    # Path to the x509 CA(s) file used to verify and authenticate client
-    # communication with the encrypted stream. This file can change and CRI-O will
-    # automatically pick up the changes within 5 minutes.
-    stream_tls_ca = ""
-
-    # Maximum grpc send message size in bytes. If not set or <=0, then CRI-O will default to 16 * 1024 * 1024.
-    grpc_max_send_msg_size = 16777216
-
-    # Maximum grpc receive message size. If not set or <= 0, then CRI-O will default to 16 * 1024 * 1024.
-    grpc_max_recv_msg_size = 16777216
-
-    # The crio.runtime table contains settings pertaining to the OCI runtime used
-    # and options for how to set up and manage the OCI runtime.
     [crio.runtime]
-
-    # default_runtime is the _name_ of the OCI runtime to be used as the default.
-    # The name is matched against the runtimes map below.
-    default_runtime = "runc"
-
-    # If true, the runtime will not use pivot_root, but instead use MS_MOVE.
-    no_pivot = false
-
-    # Path to the conmon binary, used for monitoring the OCI runtime.
-    # Will be searched for using $PATH if empty.
     conmon = "/usr/libexec/crio/conmon"
-
-    # Cgroup setting for conmon
-    conmon_cgroup = "pod"
```

Comment:
I'm pretty sure we still want this, the default is system.slice I think

Reply:
ah, I missed it, thanks! Fixed